test/name-mapping/custom/driver.ts

```diff
@@ -0,0 +1,30 @@
+import { driver, util } from '@smartlyio/oats';
+import * as process from 'process';
+
+process.chdir(__dirname);
+const nameMapper = (name: string, kind: util.NameKind) => {
+  if (kind === 'value') {
```

Comment:
even though there is a type it would be nice to have `else if kind === 'shape' ... else assert(false)` or something like that to ensure that we get the correct kind values.